lib/resolver.fallback.js

```diff
@@ -100,6 +102,30 @@ export default class PodletClientFallbackResolver {
             timeout: outgoing.timeout,
             method: 'GET',
             headers,
+            onInfo({ statusCode, headers }) {
+                if (statusCode === 103) {
+                    const parsedAssetObjects = parseLinkHeaders(headers.link);
+
+                    const scriptObjects = parsedAssetObjects.filter(
+                        (asset) => asset instanceof AssetJs,
+                    );
+                    const styleObjects = parsedAssetObjects.filter(
+                        (asset) => asset instanceof AssetCss,
+                    );
+                    // set the content js asset fallback objects
+                    // @ts-expect-error internal property
+                    outgoing.manifest._js = filterAssets(
+                        'fallback',
+                        scriptObjects,
+                    );
+                    // set the fallback css asset fallback objects
+                    // @ts-expect-error internal property
+                    outgoing.manifest._css = filterAssets(
+                        'fallback',
+                        styleObjects,
+                    );
+                }
+            },
```

Comment:
this onInfo block is called when the fallback is fetched and the podlet sends back asset early hints. The js and css received is cached for later use whenever a content route fetch fails.

Reply:
Same as on the content; I think we might need some kinda guard that only accepts one onInfo and 103 http event here.